frontend-web/webclient/app/UCloud/FileCollectionsApi.tsx

```diff
@@ -63,6 +63,8 @@ export interface FileCollectionSupport extends ProductSupport {
         aclSupported?: boolean;
         aclModifiable?: boolean;
 
+        shareSupport?: boolean;
```

Comment:
Could you create an issue about adding backend support for this?

Reply:
Yes